Lessen_6_hw/task_1.js

```diff
@@ -0,0 +1,24 @@
+/*
+1. Создайте функцию getEmployeeInfo()
+2. В функции создайте массив имен сотрудников 5шт (Имена придумать самим)
+3. В функции создайте массив зарплат сотрудников 5 шт(цифры придумать самим)
+4. Функция должна принимать 1 аргумент - имя сотрудника
+5. Функция должна возвращать новый массив, где первый элемент - имя сотрудника, второй - его зарплата
+6. Для поиска ответа функции нужно найти индекс сотрудника в массиве имен. Зарплату взять с ТЕМ ЖЕ индексом что и имя
+7. Для возврата из функции создайте массив, методом .push поместите в него имя и зарплату, и через return верните созданный массив
+8. Если такое имя сотрудника в массиве не найдется - вернуть null
+*/
+function getEmployeeInfo(name) {
+	const nameArr = ['Татьяна', 'Евгений', 'Георгий', 'Анна', 'Оксана']
+	const salaryArr = [1500, 10000, 2000, 1200, 5000]
+	const index = nameArr.indexOf(name)
+	let nameSalaryArr = []
+	nameSalaryArr.push(nameArr[index], salaryArr[index])
+	if (nameArr.indexOf(name) != -1) {
+		return nameSalaryArr
+	} else {
+		return null
+	}
```

Comment:
Return `null` right away after the search
Otherwise you're pushing `undefined` values to result array
```suggestion
	const index = nameArr.indexOf(name);
        if (index === -1) return null;
        return [nameArr[index], salaryArr[index]];
```